packages/playground-ui/src/components/ui/elements/entry-list/entry-list-item.tsx

```diff
@@ -3,20 +3,20 @@ import { Column, getColumnTemplate } from './shared';
 
 export function EntryListItem({
   item,
-  selectedItem,
+  selectedItemId,
   onClick,
   children,
   columns,
+  isLoading,
```

Comment:
isLoading can become isDisabled in this case I think. isLoading for an "ClickableEntry" means nothing while disabled does 😊